packages/x-components/src/components/decorators/__tests__/bus.decorators.spec.ts

```diff
@@ -106,8 +106,8 @@ describe('testing @XOn decorator', () => {
   });
 
   it('subscribes to a defined array of events', () => {
-    component.vm.$x.emit('UserClickedOpenX');
-    component.vm.$x.emit('UserClickedCloseX');
+    component.vm.$x.emit('UserClickedOpenEventsModal');
```

Comment:
I'm not sure why do we need to change this events, I mean you're changing the events of the close and open, but here you're testing the decorator.

I'll leave this file as it was before.

Reply:
Because the other events do not longer exist.